tests/suite/test_suite.py

```diff
@@ -58,6 +58,21 @@
 ]
 suites["reconfiguration"] = suite_reconfiguration
 
+# This suite tests that the service supports nodes on different curves, and
```

Comment:
This seems like quite an expensive way to test this. Could we add this to one of the existing test suite for example, e.g. `rekey_recovery`?